ci-operator/config/openshift/library/openshift-library-master.yaml

```diff
@@ -1,13 +1,13 @@
 base_images:
   base:
-    name: python
-    namespace: openshift
-    tag: "3.6"
+    name: "4.6"
+    namespace: ocp
+    tag: base
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.11
+    tag: golang-1.13
```

Comment:
this looks consistent @coreydaley with what we do in our operators .... I think it is OK ... will look at the rehearsals in a sec